ui/desktop/app/helpers/time-remaining.js

```diff
@@ -0,0 +1,44 @@
+import Helper from '@ember/component/helper';
+import { inject as service } from '@ember/service';
+
+export default class extends Helper {
+  // =services
+  @service clockTick;
+  @service intl;
+
+  compute([expiration_time]) {
```

Comment:
For [my comment below](https://github.com/hashicorp/boundary-ui/pull/2012/files#r1406864854) I do think, is best if this helper receives: `creation_time` and `expiration_time` and do the math with these two.

Also, this approach will give make this helper reusable everywhere we need to give time remaining, as long you pass the created time and the expiration time.